smart-contracts/test/helpers/index.js

```diff
@@ -0,0 +1,5 @@
+const errors = require('./errors')
```

Comment:
Hum, this file does not add much? Why don't we just include the errors file directly?

Reply:
oh it is a common pattern with the other PR that are ongoing (to import functions from a common ./helpers root)

Reply:
actually was removed by merging